index.js

```diff
@@ -49,51 +47,28 @@ if (!config) {
   throw new Error('you did\'t define any krabby reports. great job.');
 }
 
-tests = config.tests.map(function(test) {
-  var testName;
-  var config = {};
+var instantiatePlugins = function(plugins, path, rootMethod) {
+  return plugins.map(function(plugin) {
+    var pluginConfig = _.isString(plugin) ? { name: plugin } : plugin;
```

Comment:
Previously config was only passed if the json has specific config. By passing it always we no longer need to do that hacky name requirement from #28. w00t!